src/components/Navbar.jsx

```diff
@@ -2,10 +2,17 @@ import React from 'react';
 import { NavLink } from 'react-router-dom';
 
 const Navbar = () => (
-  <nav>
-    <NavLink to="/">Home</NavLink>
-    <NavLink to="/categories">Categories</NavLink>
-  </nav>
+  <>
+    <nav>
+      <h1>Bookstore CMS</h1>
+      <div>
+        <ul>
+          <li><NavLink to="/" className="nav-link">Home</NavLink></li>
+          <li><NavLink to="/categories" className="nav-link">Categories</NavLink></li>
+        </ul>
```

Comment:
- [ ] On the provided design these 2 links should be on the left side
your design

<img width="1440" alt="Screenshot 2023-08-04 at 20 59 11" src="https://github.com/MasumaJaffery/bookstore-reactredux/assets/44745855/6fa57e1b-4e7e-4865-93cf-e16f041f3254">

[Zeplin](https://zeplin.io/)
<img width="1440" alt="Screenshot 2023-08-04 at 21 01 16" src="https://github.com/MasumaJaffery/bookstore-reactredux/assets/44745855/d58fecae-c140-4f68-b2b7-a0a780b79afd">

on the right side you should add the `icon`

Reply:
- [ ] The update progress button should have this color 
`#0290ff`
- [ ] Please add the percentage progress as well
<img width="717" alt="Screenshot 2023-08-04 at 21 08 58" src="https://github.com/MasumaJaffery/bookstore-reactredux/assets/44745855/fa96f5c7-a21a-4653-a8bb-bc962c832a03">

Reply:
- These buttons that you added as placeholders should look like the given design I suggest that you use this color
`#4386bf`
<img width="379" alt="Screenshot 2023-08-04 at 21 06 39" src="https://github.com/MasumaJaffery/bookstore-reactredux/assets/44745855/3f102390-0768-442f-aca5-a15701edcdfb">